pandas/core/frame.py

```diff
@@ -4635,7 +4636,16 @@ def quantile(self, q=0.5, axis=0, numeric_only=True):
             0 <= q <= 1, the quantile(s) to compute
         axis : {0, 1, 'index', 'columns'} (default 0)
             0 or 'index' for row-wise, 1 or 'columns' for column-wise
```

Comment:
So ``.quantile`` has the implementeation in Series/DataFrame (it could be combined but that's a different issue). So the doc-string and the ``.quantile`` method go in ``core/generic/NDFrame`` and will raise a ``NotImpementedError``